spaceship/lib/spaceship/spaceauth_runner.rb

```diff
@@ -49,19 +51,26 @@ def run
 
       yaml = cookies.to_yaml.gsub("\n", "\\n")
 
-      puts("---")
-      puts("")
-      puts("Pass the following via the FASTLANE_SESSION environment variable:")
-      puts(yaml.cyan.underline)
-      puts("")
-      puts("")
-      puts("Example:")
-      puts("export FASTLANE_SESSION='#{yaml}'".cyan.underline)
+      if @output.nil?
+        puts("---")
+        puts("")
+        puts("Pass the following via the FASTLANE_SESSION environment variable:")
+        puts(yaml.cyan.underline, force: true)
+        puts("")
+        puts("")
+        puts("Example:")
+        puts("export FASTLANE_SESSION='#{yaml}'".cyan.underline)
+
+        if mac? && Spaceship::Client::UserInterface.interactive? && agree("🙄 Should fastlane copy the cookie into your clipboard, so you can easily paste it? (y/n)", true)
+          require 'open3'
+          Open3.popen3('pbcopy') { |input, _, _| input << yaml }
+          puts("Successfully copied text into your clipboard 🎨".green)
+        end
+      end
 
-      if mac? && Spaceship::Client::UserInterface.interactive? && agree("🙄 Should fastlane copy the cookie into your clipboard, so you can easily paste it? (y/n)", true)
-        require 'open3'
-        Open3.popen3('pbcopy') { |input, _, _| input << yaml }
-        puts("Successfully copied text into your clipboard 🎨".green)
+      unless @output.nil?
```

Comment:
lets just make this an `else` of the `if @output.nil?` then

Reply:
OK